app/src/Model/Entity/User.php

```diff
@@ -43,4 +44,23 @@ class User extends Entity
     protected $_hidden = [
         'password',
     ];
+
+    /**
+     * Set and hash the user password.
+     *
+     * This method is used to set and hash the user's password before saving it to the database.
+     * It checks the length of the provided password and, if it's not empty, hashes the password
+     * using the default password hasher.
+     *
+     * @param string $password The user's plaintext password.
+     * @return string|null The hashed password or null if the provided password is empty.
+     */
+    protected function _setPassword(string $password): ?string
+    {
+        if (strlen($password) > 0) {
+            return (new DefaultPasswordHasher())->hash($password);
```

Comment:
Since we hash the password, we shall fix the seed to hash the password as well. https://github.com/ArielGalvez/tutorial-cakephp/blob/f63439f20b049ecd6040fa23e2d417e8293ec994/app/config/Seeds/UsersSeed.php#L28

Reply:
applied